packages/flutter_test/lib/src/binding.dart

```diff
@@ -936,41 +936,83 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
       );
     }());
 
+    final List<Timer> timerCache = <Timer>[];
+    final List<Timer> repeatingTimerCache = <Timer>[];
+    int microtasks = 0;
+    DateTime latestMicrotaskCreationTime;
     final Zone realAsyncZone = Zone.current.fork(
       specification: ZoneSpecification(
         scheduleMicrotask: (Zone self, ZoneDelegate parent, Zone zone, void f()) {
-          Zone.root.scheduleMicrotask(f);
+          print('1 with ${self.hashCode}');
+          microtasks++;
+          latestMicrotaskCreationTime = DateTime.now();
+          Zone.root.scheduleMicrotask(() async {
+            f();
+            microtasks--;
+            print('Done');
```

Comment:
debug statement

Reply:
Will be removed once we agree on implementation :-)